packages/vitest/src/runtime/entry-vm.ts

```diff
@@ -41,7 +41,9 @@ export async function run(files: string[], config: ResolvedConfig, executor: Vit
   if (config.chaiConfig)
     setupChaiConfig(config.chaiConfig)
 
+  const diffConfig = await loadDiffConfig(config, executor)
```

Comment:
Can't you do this inside `resolveTestRunner`?

Reply:
Thank you for reviewing, already updated.